sdk/eventhubs/azure_messaging_eventhubs/benches/benchmarks.rs

```diff
@@ -27,7 +27,9 @@ fn send_batch_benchmark(c: &mut Criterion) {
     setup();
 
     // Check if the environment variable is set thus allowing the benchmarks to run
-    if azure_core_test::TestMode::current().unwrap() != azure_core_test::TestMode::Live {
+    if azure_core_test::TestMode::current().unwrap_or(azure_core_test::TestMode::Playback)
```

Comment:
This is fine, but `TestMode` derives `Default` with `Playback` as the default, so you can just do `unwrap_or_default()`.